include/event_manager.h

```diff
@@ -147,52 +144,142 @@ extern "C" {
  */
 #define EVENT_SUBMIT(event) _event_submit(&event->header)
 
+/**
+ * @brief Register event hook after the event manager is initialized
```

Comment:
Nit: period is missing at the end here and at the end of several other "@brief"s.

Reply:
Fixed